.github/workflows/readme-linter.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-          go-version: '1.22.1'
+          go-version: '1.22.3'
```

Comment:
This is already done as well

Reply:
You are right here! No action needed here.